src/components/AFEpage/SmallScreenAbbreviate.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+export default function SmallScreenAbbreviate(props) {
+  const text = props.text || "";
+  const isActive = props.isActive || false;
```

Comment:
It's not necessary to coerce `props.isActive` to a Boolean here when it's `undefined` or some other falsy value.  But also, I think it'd be better if this Component decides whether to show an abbreviation on its own based on the viewport width.  Perhaps, it could be extended to be supplied with a breakpoint width and default to the small one.

```suggestion
  const isActive = props.isActive;
```